client/src/models/years.js

```diff
@@ -1,5 +1,23 @@
 import { run_template } from "./text.js";
 
+const fiscal_year_to_year = (fy_string) =>
+  _.chain(fy_string).split("-").first().toNumber().value() || null;
+
+const year_to_fiscal_year = (year) => {
+  if (year) {
+    const second_year = (() => {
+      if (window.lang === "en") {
+        return year + 1;
+      } else {
+        return (year + 1).toString().substring(2);
+      }
+    })();
```

Comment:
Fair point! No particular reason, I've just been using IIFs a lot when declaring dynamic-valued `const`s. Probably because it's what I recommended to the CO-OPs to do, since the IIF approach is more flexible. I'll switch to the immediate-lookup\* approach here for brevity.

\* is that a good name for this pattern? Just picked it